vizro-ai/mkdocs.yml

```diff
@@ -3,18 +3,16 @@ site_url: https://vizro-ai.readthedocs.io/en/latest/
 nav:
   - Vizro-AI: index.md
   - Get started:
-      - Quickstart: pages/tutorials/quickstart.md
+      - Install Vizro-AI: pages/user-guides/install.md
+      - A first dashboard: pages/tutorials/quickstart.md
       - Explore Vizro-AI: pages/tutorials/explore-vizro-ai.md
-  - User Guides:
-      - Fundamentals:
-          - Installation: pages/user-guides/install.md
```

Comment:
Where has the installation guide gone e.g.? I think this belongs to a "how to" guide no?

Reply:
I've moved the page in the layout to the "Get started" section as I did for `vizro-core`. I agree that it is a how-to guide in that it's shows how to install, but I didn't like the user jumping from quickstart straight away into how-to, and decided to bring installation directly into the get started instructions (as, after all, they can't get started until they've done the install).

https://vizro--381.org.readthedocs.build/projects/vizro-ai/en/381/pages/user-guides/install/